lib/apollo-federation/schema.rb

```diff
@@ -38,7 +38,11 @@ def federation_sdl(context: nil)
       end
 
       def link_namespace
-        @link[:as]
+        if @link
```

Comment:
Would you mind updating this implementation to use `GraphQL::Schema::FindInheritedValue`?